helmfile.d/0410.grafana.yaml

```diff
@@ -102,3 +133,27 @@ releases:
             hosts:
               ### Required: GRAFANA_HOSTNAME;
               - '{{ env "GRAFANA_HOSTNAME" }}'
+
+
+#######################################################################################
+## Grafana DB                                                                        ##
+#######################################################################################
+
+- name: "grafana-db"
+  namespace: "monitoring"
+  labels:
+    chart: "mysql"
+    repo: "stable"
+    component: "monitoring"
+    namespace: "monitoring"
+    vendor: "kubernetes"
+    default: "false"
+  chart: "stable/mysql"
+  version: "0.10.1"
+  wait: true
+  values:
+  - mysqlUser: '{{ env "GRAFANA_DB_USER" | default "grafana" }}'
+    mysqlPassword: '{{ env "GRAFANA_DB_PASSWORD" | default "grafana"}}'
+    mysqlRootPassword: '{{ env "GRAFANA_DB_ROOT_PASSWORD" | default "grafana"}}'
+    mysqlDatabase: '{{ env "GRAFANA_DB_DATABASE_NAME" | default "grafana"}}'
```

Comment:
For consistency, seems like this should be just `GRAFANA_DB_DATABASE` since in the other variables you strop `mysql`, and prefix with `GRAFANA_DB`

Reply:
@osterman then `GRAFANA_DB_NAME` is better?
Because `GRAFANA_DB_DATABASE` is short of `GRAFANA_DATABASE_DATABASE`